format/BUILD.bazel

```diff
@@ -61,6 +61,15 @@ alias(
     }),
 )
 
+alias(
```

Comment:
How do we override this? I'd rather not have out of sync Tf versions, and need to download it twice

Reply:
It's in the readme, user just has to declare theirs first in workspace with a matching name

Reply:
And in silo we already do